### PR TITLE
Add Fertile Forest plugin.

### PR DESCRIPTION
**Pull request by StewEucen**
@dereuromark 
I made plugin for Fertile Forest Behavior.
It is the fifth model for hierarchical data in a relational database.

Maybe, it will change our treasure. 

https://github.com/StewEucen/fertile-forest-cakephp-plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ Focus is on the specific task.
 - [Muffin/Trash plugin](https://github.com/usemuffin/trash) - Soft-delete behavior for CakePHP.
 - [PersistRelatedData plugin](https://github.com/riesenia/persist-related-data) - Behavior for persisting selected fields of related models.
 - ~~[Datasources plugin](https://github.com/cakephp/datasources) - Lots of different datasources.~~
+- [StewEucen/Fertile Forest plugin](https://github.com/StewEucen/fertile-forest-cakephp-plugin) - Behavior for Fertile Forest the model for hierarchical data in a daabase.
```
Comment (dereuromark) [inline README.md]: Typo "database".

**Comment by dereuromark** [comment]
Tests would be useful, and travis CI along with it.

**Comment by StewEucen** [comment]
@dereuromark 
I am sorry, I made some mistakes.
The branch that I pushed did not contains upstream commits.

I make new branch with last upstream of master, and send pull request.
X add-fertile-forest-plugin
O add-FertileForest-plugin

File: README.md [hunk 2]
```diff
@@ -218,6 +218,7 @@ Focus is on the specific task.
 - [Muffin/Trash plugin](https://github.com/usemuffin/trash) - Soft-delete behavior for CakePHP.
 - [PersistRelatedData plugin](https://github.com/riesenia/persist-related-data) - Behavior for persisting selected fields of related models.
 - ~~[Datasources plugin](https://github.com/cakephp/datasources) - Lots of different datasources.~~
+- [StewEucen/Fertile Forest plugin](https://github.com/StewEucen/fertile-forest-cakephp-plugin) - Behavior for Fertile Forest the model for hierarchical data in a database.
```
Comment (dereuromark) [inline README.md]: the name is not correct. please see the others how the name should be.

**Pull request closed without merging**